client/src/css/Grace.css

```diff
@@ -75,6 +85,22 @@
   margin-bottom: 20px;
 }
 
+.links {
+  color: #fdf1e8;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  background-color: #10435c;
+  margin-top: -50px;
+  position: relative;
+  padding-left: 60px;
+  padding-right: 60px;
+}
```

Comment:
I agree with Ashwin that you should make your classNames more specific. You have .link and .links, but it isn't immediately obvious what the difference between these names are. At the same time, it's easy to make a mistake and use the wrong className since the names are so similar